### ISSUE TITLE
RZUpdate: E131:Der Wert darf nicht NULL sein.

### ISSUE THREAD

**Issue by Nama** [rzander/ruckzuck#63]
Hi,

I wanted to switch to Ruckzuck from Ninite, but I have some trouble running RZUpdate.

Using 1.7.1.1

If updates are detected, this is the output:
```
 .\RZUpdate.exe /Update
Detecting updates...... done.
1 updates found.
E131:Der Wert darf nicht NULL sein.
Parametername: source
```

But Ruckzuck.exe is able to update.

**Comment by rzander** [comment]
Ouch, thats a bug if no RZUpdate.exe.config file is found...
Is this fixed if you place the following RZUpdate.exe.config File (remove the .txt extension) in the same Dir as RZUpdate.exe ?
[RZUpdate.exe.config.txt](https://github.com/rzander/ruckzuck/files/3795586/RZUpdate.exe.config.txt)

**Comment by rzander** [comment]
Released a fixed Version V1.7.1.1...

**Comment by Nama** [comment]
I just ran Ninite. I have to wait for Updates^^
